File: mobile/lib/constants/resposta_sim_nao.dart
enum RespostaSimNao {
  sim('SIM', 'Sim'),
  nao('NAO', 'Não');

  const RespostaSimNao(this.apiValue, this.rotulo);

  final String apiValue;
  final String rotulo;

  static RespostaSimNao? fromApi(String? value) {
    if (value == null) return null;
    for (final r in values) {
      if (r.apiValue == value) return r;
    }
    return null;
  }
}

File: backend/src/lib/resposta-sim-nao.ts
import { RespostaSimNao } from "@prisma/client";

export const RESPOSTAS_SIM_NAO = ["SIM", "NAO"] as const satisfies readonly RespostaSimNao[];

export const ROTULO_RESPOSTA_SIM_NAO: Record<RespostaSimNao, string> = {
  SIM: "Sim",
  NAO: "Não",
};
